tests/urlpatterns_reverse/named_urls_conflict.py

```diff
@@ -0,0 +1,18 @@
+from django.conf.urls import url
+
+from .views import empty_view
+
+urlpatterns = [
+    # no args
```

Comment:
The three comments can be removed. Leaving a blank line between the "test sets" is OK.